app/models/crop.rb

```diff
@@ -120,12 +120,7 @@ def default_scientific_name
   # later we can choose a default photo based on different criteria,
   # eg. popularity
   def default_photo
-    # most recent photo
-    return photos.order(created_at: :desc).first if photos.any?
-
-    # Crop has no photos? Look for the most recent harvest with a photo.
-    harvest_with_photo = Harvest.where(crop_id: id).joins(:photos).order('harvests.id DESC').limit(1).first
-    harvest_with_photo.photos.first if harvest_with_photo
+    first_photo(:plantings) || first_photo(:harvests) || first_photo(:seeds)
```

Comment:
Should this fall back to placeholder on the end, or is it okay not to?

Reply:
It will fall back on placeholder. That's done in the helpers.